graphjet-core/src/main/java/com/twitter/graphjet/hashing/SmallArrayBasedLongToDoubleMap.java

```diff
@@ -171,6 +171,24 @@ public boolean trim(int inputCapacity) {
   }
 
   /**
+   * @param key the input capacity that clients want to trim the map to
+   * @return true if the key is in the Map.
+   */
+  public boolean contains(long key) {
```

Comment:
The javadoc for this class says that there is no get function, so perhaps add a warning in the doc for this function to discourage wider use since it scans linearly.